src/Diagnostics.RuntimeHost/Controllers/DiagnosticControllerBase.cs

```diff
@@ -244,13 +244,13 @@ protected async Task<IActionResult> ExecuteQuery<TPostBodyResource>(TResource re
                     string[] utterances = null;
                     if (jsonBody.DetectorUtterances != null && invoker.EntryPointDefinitionAttribute.Description.ToString().Length > 3)
                     {
-                        utterances = JsonConvert.DeserializeObject<string[]>(jsonBody.DetectorUtterances);
-                        string description = invoker.EntryPointDefinitionAttribute.Description.ToString();
-                        var resourceParams = _internalApiHelper.GetResourceParams(invoker.ResourceFilter);
-                        var searchUtterances = await _searchService.SearchUtterances(runtimeContext.OperationContext.RequestId, description, utterances, resourceParams);
-                        string resultContent = await searchUtterances.Content.ReadAsStringAsync();
                         try
                         {
+                            utterances = JsonConvert.DeserializeObject<string[]>(jsonBody.DetectorUtterances);
+                            string description = invoker.EntryPointDefinitionAttribute.Description.ToString();
+                            var resourceParams = _internalApiHelper.GetResourceParams(invoker.ResourceFilter);
+                            var searchUtterances = await _searchService.SearchUtterances(runtimeContext.OperationContext.RequestId, description, utterances, resourceParams);
```

Comment:
Any true errors with the service call are going to get squashed in production, which is more important than being able to run the service locally. We may want to wire in the configuration setting for "local server" and skip the service if it's set